webui/react/src/pages/ExperimentDetails/ExperimentTrials.tsx

```diff
@@ -166,6 +166,19 @@ const ExperimentTrials: React.FC<Props> = ({ experiment, pageRef }: Props) => {
       );
     };
 
+    const logRetentionDaysRenderer = (_: string, record: TrialItem): React.ReactNode => {
+      const daysExist = record.logRetentionDays ? true : false;
+      return (
+        <span>
+          {!daysExist
+            ? ''
+            : record.logRetentionDays === -1
+              ? 'Forever'
+              : `${record.logRetentionDays} days`}
+        </span>
```

Comment:
its a bit longer but, i think its readable compared to nested ternary operators

```suggestion
      const logRetentionDays = record.logRetentionDays; //  to take advantage of ts type
      if (logRetentionDays === undefined) { // only `logRetentionDays` causes unwanted behavior when the value is 0
        return <span>-</span>; // or null depends of what you wanna show
      }
      return (
        <span>
          {logRetentionDays === -1
            ? 'Forever'
            : `${logRetentionDays} ${pluralizer(logRetentionDays, 'day')}`} // plural 
        </span>
      );
```

Reply:
done!